src/Controller/OgAutocompleteMatcher.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
@jludwig  we should remove this file completely, right? `OgAutocompleteController` lives in another file.